packages/yourloops/pages/caregiver/index.tsx

```diff
@@ -134,15 +112,12 @@ const CaregiverPage = (): JSX.Element => {
   } else if (session === null || sharedUsersState.sharedUsers === null || loading) {
     content = <CircularProgress disableShrink className={classes.loadingProgress} />;
   } else {
-    content = <CaregiverRoutes />;
+    content = children;
```

Comment:
`content` could be directly initialized with `children` and thus avoid having an `else` case